### ISSUE TITLE
selenium 4.3.0 removes Opera support - now causes an ImportError in selenium-requests

### ISSUE THREAD

**Issue by jprouty** [cryzed/Selenium-Requests#51]
Example stack trace:
```
    from seleniumrequests import Chrome
  File "/Users/jeff/repos/mint-amazon-tagger/dev_venv/lib/python3.8/site-packages/seleniumrequests/__init__.py", line 8, in <module>
    from selenium.webdriver import Opera as _Opera
ImportError: cannot import name 'Opera' from 'selenium.webdriver' (/Users/jeff/repos/mint-amazon-tagger/dev_venv/lib/python3.8/site-packages/selenium/webdriver/__init__.py)
```

Commit that removed Opera from selenium is [here - 10 days ago](https://github.com/SeleniumHQ/selenium/commit/17907d38c8e153f9d02ba4e294bce7843d4d6a41).

**Comment by cryzed** [comment]
Fixed in #50 and subsequent commits. Thanks for the heads-up!

**Comment by jprouty** [comment]
Perfect, thanks for the new pip version. I've verified on my end that everything is groovy again. Thanks for this great library!
